### PR TITLE
Feature/vacation event

### PR DESCRIPTION
**Pull request by serjKim**
Save and cancel vacation

### CODE REVIEW

File: mobile/src/calendar/event-dialog/edit-vacation-event-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        this.props.closeDialog();
+    }
+
+    public get text(): string {
+        const { intervals: { vacation } } = this.props;
+
+        const startDate = vacation.calendarEvent.dates.startDate.format(eventDialogTextDateFormat);
+        const endDate = vacation.calendarEvent.dates.endDate.format(eventDialogTextDateFormat);
+
+        return `Your vacation has started on ${startDate} and will be compelete on ${endDate}.`;
```
Comment (santee) [inline mobile/src/calendar/event-dialog/edit-vacation-event-dialog.tsx]: 'starts' and 'completes'

File: mobile/src/calendar/event-dialog/event-dialog.tsx [hunk 2]
```diff
@@ -30,6 +33,13 @@ export class EventDialog extends Component<EventDialogProps> {
                 return <EditSickLeaveEventDialog />;
             case EventDialogType.ProlongSickLeave:
                 return <ProlongSickLeaveEventDialog />;
+
+            case EventDialogType.RequestVacaltion:
+                return <RequestVacationEventDialog />;
+            case EventDialogType.ConfirmStartDateVacation:
+                return <ConfirmVacationEventDialog />;
+            case EventDialogType.EditVacaltion:
```
Comment (santee) [inline mobile/src/calendar/event-dialog/event-dialog.tsx]: typo

File: mobile/src/calendar/event-dialog/event-dialog.tsx [hunk 3]
```diff
@@ -30,6 +33,13 @@ export class EventDialog extends Component<EventDialogProps> {
                 return <EditSickLeaveEventDialog />;
             case EventDialogType.ProlongSickLeave:
                 return <ProlongSickLeaveEventDialog />;
+
+            case EventDialogType.RequestVacaltion:
```
Comment (santee) [inline mobile/src/calendar/event-dialog/event-dialog.tsx]: typo

File: mobile/src/calendar/calendar-actions-button-group.tsx [hunk 4]
```diff
@@ -32,16 +36,26 @@ export class ActionsButtonGroupImpl extends Component<ActionButtonGroupProps & A
 
         return (
             <View style={calendarActionsStyles.container}>
-                <VacationActionButton interval={intervalsBySingleDaySelection.vacation} disabled={this.props.disableActionButtons} />
+                <VacationActionButton
```
Comment (santee) [inline mobile/src/calendar/calendar-actions-button-group.tsx]: do we check that user can't change dates in approved vacation? Only cancel it.

Comment (serjKim) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: yes, we do, but I thought to add it with edit next PR

Comment (santee) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: alright

Comment (santee) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: do you want to add a separate dialog for that, perhaps?

Comment (serjKim) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: yes

**Pull request merged**